dev/kube/ig-tools/snap-resources.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Do you think this + max memory could live in kubecf tools?
Other projects could benefit.

Reply:
Yes, it could ... And agree, it should.

Reply:
Note: The `yaml-memory.sh` is specific to `kubecf` and its pods/containers. Should stay here. Also a reason to keep it separate from the `max memory`, which is generic.

Reply:
See https://github.com/cloudfoundry-incubator/kubecf-tools/pull/8

Reply:
See f96ab53a9b3968104baa30518261ab915377866b for shift of the generics.